recipes/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "torsiondrive" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "5fe4a238ffeaf35159bbe9556a37134475aeac0fd40bb69b8f9d538c46f2fc25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - torsiondrive-launch = torsiondrive.launch:main
+    - torsiondrive-api = torsiondrive.td_api:main
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - {{ pin_compatible('numpy', lower_bound='1.11') }}
+    - python
+
+test:
+  imports:
+    - pytest
+    - torsiondrive
+  requires:
+    - pytest
+  commands:
+    - py.test
+
+about:
+  home: https://github.com/lpwgroup/torsiondrive
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
You will need to manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
The license file named `LICENSE` is included in the tarball. I have fixed the typo here. Is it good?